clang-tools-extra/test/clang-tidy/checkers/readability/avoid-return-with-void-value.cpp

```diff
@@ -12,14 +12,18 @@ void f2() {
     return f1();
     // CHECK-MESSAGES: :[[@LINE-1]]:5: warning: return statement within a void function should not have a specified return value [readability-avoid-return-with-void-value]
     // CHECK-MESSAGES-LENIENT: :[[@LINE-2]]:5: warning: return statement within a void function should not have a specified return value [readability-avoid-return-with-void-value]
+    // CHECK-FIXES: f1(); return;
```

Comment:
Is an unnecessary `return` okay? Otherwise, this would need an analysis that goes beyond the scope of this rule.